docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -0,0 +1,51 @@
+import type {ReactNode} from 'react';
+import clsx from 'clsx';
+import Heading from '@theme/Heading';
+import styles from './styles.module.css';
+
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: ReactNode;
+};
+
+const FeatureList: FeatureItem[] = [
+  // {
+  //   title: 'Easy to Use',
+  //   Svg: require('@site/static/img/undraw_docusaurus_mountain.svg').default,
+  //   description: (
+  //     <>
+  //       Docusaurus was designed from the ground up to be easily installed and
+  //       used to get your website up and running quickly.
+  //     </>
+  //   ),
+  // },
```

Comment:
Same with this, if we're not using it, we probably don't need to keep it

Reply:
I do plan to use it later, and unlike the sidebar config object structure I don't think it's documented on the Docusaurus site. So I think I'm going to keep the commented-out example data structure for future reference, then delete it when we do a nice home page.